File: tests/fixtures/sample.lua
-- Line comment
--[[ Block comment ]]
local x = 42 -- inline

local s = "hello -- not a comment"
local s2 = 'world -- not a comment'
local s3 = [[ long string -- not a comment ]]
local s4 = [=[ long string ]=] not end -- still string ]=]

--[=[
Multi-line
long comment
]=]
print(x)

File: tests/fixtures/sample.js
// Line comment
/* Block comment */
const x = 42; // inline

const str = "hello // not a comment";
const str2 = 'world /* not a comment */';
const tmpl = `template // ${x + /* not stripped */ 1} end`;
const re = /pattern/gi;
const y = a / b; // division, not regex
const re2 = /[/]/; // regex with slash in class

/* Multi-line
   block
   comment */
console.log(str);
